procfile/README.md

```diff
@@ -42,3 +42,13 @@ Specifies the command that should be run when executing this process
 ```yaml
 command: ./bin/web
 ```
+
+**Expose**
+
+Specifies the exposure settings for the process. This allows you to expose a tcp/http/https/ssl service on any process in the application.
+
+```yaml
+expose:
+  type: tcp // Possible options are tcp/http/https/ssl
```

Comment:
Actually, I need to check if this would even be possible with ECS. To do this, I think we'd need to allocate an instance port per container port (e.g. port 8080 and port 8081 get different instance ports). Last time I checked, you could only add 1 LoadBalancer object when creating an ECS service, and the load balancers can't be updated once the ecs service is created.

Maybe the format above is good, but we just need some validations for ECS/ELB limitations.